### PR TITLE
Stratconn 2671 optimizely public document

### PR DESCRIPTION
**Pull request by hvardhan-unth**
### Proposed changes

This PR adds public document for Optimizely Feature Experimentation (Actions).
JIRA ticket link:- https://segment.atlassian.net/browse/STRATCONN-2671

### Merge timing
When should this get merged/published?
- ASAP once approved

### CODE REVIEW

File: src/_data/catalog/destination_categories.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # AUTOGENERATED FROM PUBLIC API. DO NOT EDIT
```
Comment (varadarajan-tw) [inline src/_data/catalog/destination_categories.yml:1]: Are the files under `_data/catalog` relevant to this change? If not, can we remove them?

Comment (forstisabella) [reply src/_data/catalog/destination_categories.yml]: Hi - we update our catalog before every deployment (Tuesdays & Thursdays) and so those files were out of sync with our current `develop` branch and were not in the scope of this PR.

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+To connect the Optimizely Feature Experimentation destination:
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Search for **Optimizely Feature Experimentation** in the Destinations Catalog, and select the destination.
+3. Click **Configure Optimizely Feature Experimentation (Actions)** in the top-right corner of the screen.
+4. Select the source that will send data to Optimizely Feature Experimentation and follow the steps to name your destination.
+5. On the **Settings** tab, input your `datafile` URL and your Account Id. Toggle “Enable Destination” on and click  **Save Changes**.
+6. Navigate to the **Mappings** tab, click **New Mapping**, and select **Track Event**.
+7. Under Select mappings, Select the event key from the dropdown or input the event manually as the  "event" and select the other mappings. Click **Save** and toggle to enable the mapping.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md:37]:
```suggestion
7. Under Select mappings, select the event key from the dropdown or input the event manually as the "event" and select the other mappings. Click **Save** and toggle to enable the mapping.
```

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+     * **Note:** The conversion event will only be sent if the event key configured should match exactly as the name of an active experiment metric set up in the Optimizely dashboard.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md:38]:
```suggestion
     * **Note:** The conversion event will only be sent if the configured event key configured exactly matches the name of an active experiment metric set up in the Optimizely dashboard.
```

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+     * **Note:** The current user should activated in a running experiment with the associated metric. (Activation is expected to be done at the event source before sending track events to Segment).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md:39]: What does this say? The Segment user should activate a running experiment with the metric mentioned in the previous note?
```suggestion
     * **Note:** The current user should activate a running experiment with the associated metric before sending track events to Segment.
```

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+## GDPR Support
+Segment supports deleting/suppressing users in Optimizely Feature Experimentation using the [Segment app](/docs/privacy/user-deletion-and-suppression/). In order to do this however, you will need to create a [Personal Access Token](https://developers.optimizely.com/x/authentication/personal-token/) in Optimizely and provide it as the value of the Personal Access Token setting.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md:42]:
```suggestion
Segment supports deleting/suppressing users in Optimizely Feature Experimentation (Actions) using the [Segment app](/docs/privacy/user-deletion-and-suppression/). Before deleting/suppressing a user, create a [Personal Access Token](https://developers.optimizely.com/x/authentication/personal-token/) in Optimizely and provide it as the value of the Personal Access Token setting.
```

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+{% include components/actions-fields.html settings="true"%}
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md:44]: There are no mappings that appear when the site is rendered - is that intentional?

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md (hunk 4 above)
Comment (hvardhan-unth) [reply src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md]: Yes, The user should activate in running experiment with the metric mentioned in event.

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+[Optimizely Feature Experimentation](https://www.optimizely.com/products/experiment/feature-experimentation/){:target="_blank"} is a feature flagging and experimentation platform for websites, mobile apps, chatbots, APIs, smart devices and anything else with a network connection.
+
+With their SDK you can deploy code behind feature flags, experiment with A/B tests and use percentage deliveries to roll out or roll back flags immediately.
+
+Segment’s Optimizely Feature Experimentation (Actions) destination supports tracking of conversion events.
+Segment supports one action (event): trackEvent. [TrackEvent](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/track-event-javascript-node){:target="_blank"} sends user conversion events for active experiments. Segment sends data using Optimizely's [Event API](https://docs.developers.optimizely.com/experimentation-data/reference/post_events){:target="_blank"}.
+
+## Implementation Prerequisite
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md:16]:
```suggestion
## Prerequisites
```

File: src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+### Track
+
+Upon invocation of a Segment `track` event, Segment maps the event to an Optimizely `track` event:
+* If the Segment event name matches exactly the name of an active experiment `metric` set up in the Optimizely dashboard;
+* If the experiment `metric` is associated with a running experiment;
+* If the current user has been assigned a `userId` using Segment's `identify` method (for example,  `analytics.identify('123')`);
+* If the current user is activated in a running experiment with the associated `metric`.
+
+Segment also handles the following mapping:
+* Segment `track` event name to Optimizely `eventName`.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-feature-experimentation-cloud/index.md:50]:
```suggestion
* Segment maps `track` events to Optimizely `eventName`.
```

**Review by forstisabella — APPROVED** [review]
A couple small notes, but looks good to me!

**Pull request merged**